Packages/MIES/MIES_SweepFormula.ipf

```diff
@@ -70,6 +70,8 @@ Function FormulaParser(formula)
 	Variable jsonID = JSON_New()
 	String jsonPath = ""
 
+	formula = LowerStr(formula)
```

Comment:
note that this will convert also `AD0` to `ad0`. If your intention was to allow case-insensitive operations/function naming then this should better go directly to the executor, wrapping `Lowerstr(operation[0])` and we should not camelCase operations then.

Reply:
dd89f1802c4717af66f3eec109c4b5348d5c331c